README.md

```diff
@@ -1 +1,56 @@
-# to-do-list.
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-Do-List
+
+> This project was built as part of the Microverse JavaScript module. The goal of the project is to build a simple Todo App using Webpack and ES6 by dynamically creating the DOM elements.
+
+![screenshot](SCREENSHOT.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML5
+- CSS3
+- Bootstrap5
+- JavaScript
+- Babel
+- Webpack
+- localStorage
+
+
+## Getting Started
+
+In order to run locally this project type the following commands over the terminal in Linux or Mac or the Windows cmd console:
+
+```s
+$ git clone https://github.com/prernapriyali/to-do-list-microverse
+
+
+```
+
+After running the previous commands, to see the web application in the browser go to the `To-Do-List` folder and open the `index.html` file inside `dist` folder.
```

Comment:
- [ ] Those instructions are not complete, let's improve them a bit as described below:
  - What if the user wants to modify the code a bit, how can he build the project?
  - Please include all the steps needed to:
    1. install the dependencies of the project with npm
    2. build the project into dist
    3. run the project on the browser

Reply:
Just one more change is required here:
- Add instructions/command to build the project to Readme